test/vaadin-list-mixin.html

```diff
@@ -479,6 +478,142 @@
           expect(list._scrollerElement.scrollTop).to.be.greaterThan(0);
         });
       });
+
+      describe('Multiple', () => {
+        it('should clear selected when multiple=true', () => {
```

Comment:
Native select keeps the selected item with lowest index as its `value` in multiselection mode.
(In case we want to totally follow that functionality)